source/_posts/2017-05-19-kanban-method-in-a-consulting-company.md

```diff
@@ -0,0 +1,123 @@
+* Label - According to the task,
+* Due date - We set a maximum of two weeks to complete the tasks.
+
+The card is very important in the whole project process. That way, we measure 
+our work and set time for each stage in the design, develop and testing 
+solution phases in embedded systems projects. Additionally inside the
+card we can discuss the features and see, what happened since last time.
+
+![img](http://imageshack.com/a/img924/2586/spqDrv.png)
+
+## Workflow
+ 
+The card life begins in Backlog, tasks here have lowest priority and they are
+waiting for select to development - but it's not a rule, sometimes we add cards
+with high priority directly to ToDo column. ToDo is the queue for the most
+important column - In Progress. From there our goal is to safely move this task
+to done. By analyzing the boards we are able to find the blocker very quickly.
+In embedded systems company, we are dealing mostly with blockers in the form of
+not working or missing hardware, incompatibility of the kernel with hardware and 
+frequent changes in priorities.
+
+![kanban-flow](http://imageshack.com/a/img923/2471/u20Zzr.jpg)
+
+## Rules
+
+To make kanban work, each employee must follow these rules:
+
+* In Progress has the biggest priority, next is ToDo and Backlog,
+* Never take more than two tasks at a time,
+* Do not throw unfinished tasks for verification, because due date is comming,
+* Listen and read communication, ask questions,
+* Discuss inside cards - about problems and progress,
+* Suggest changes in places where you see such a need.
+
+These rules are only a sample but improve the whole process. We know situations
+where tasks are going fast to verification, after which it turns out that the
+task is incomplete and we need to turn it back to the left. When the checks show
+such errors, we try to first determine the degree of completion of the card, and
+then decide whether the card must return or sometimes we split cards and move 
+part of result to Verification and other part to ToDo. In our field there are 
+frequent changes, a very good flow of information is important to save 
+unnecessary work.
+
+## Pros and cons
+
+The advantages are:
+
+* Universal - can be used to visualize company overview, project or employee
+individual workflow
+* Creating a WIP (Work In Progress) limit - it prevents overproduction and 
+quickly detects bottlenecks
+* Clarity - one look and you know the whereabouts of the project
+* Stream management - systematic time and effort measurement to optimize
+processes.
+
+ And cons:
+
+* Need for continuous control
+* Commitment of each employee
+* Incorretly managed will turn into a battlefield where employees struggle, who
+has done more and create a nasty mess on the work cards.
+
+## Summary
+
+Kanban is designed for companies looking for ways to optimize task allocation
+and for teams that are overloaded with tasks. It works because introducing lower

```

Comment:
Sentence syntax is wrong "It works well, because it introduces lower task settlements and by the way it adds very good tools for work progress measuring and overproduction managing." Do not mix present continuous and present simple this way. If You start with "introducing" then it should be "adding" and if You have "adds" then write "introduces". Stick to one either continuous or simple.